### PR TITLE
refresh collection detail pages (bug 1103202/1103304)

### PR DESCRIPTION
**Pull request by ngokevin**
PR before swapping the branch to point to feed: https://github.com/mozilla/fireplace/pull/890

screen shot 2015-01-12 at 11 50 02 am
screen shot 2015-01-12 at 11 49 56 am
screen shot 2015-01-12 at 11 50 19 am
screen shot 2015-01-12 at 11 50 11 am
screen shot 2015-01-12 at 11 50 42 am
screen shot 2015-01-12 at 11 50 27 am

### CODE REVIEW

**Comment by ngokevin** [comment]
Going to get rid of the anchor link on the collection tile on mobile.

File: src/media/css/app-list.styl [hunk 1]
```diff
@@ -41,9 +41,11 @@ $desktop-tile-padding = 20px 50px;
     margin-top: 30px;
 
     li:not(.loadmore) {
-        border-bottom: 1px solid $filters-border;
         padding-top: 15px;
     }
+    li:not(.loadmore):not(:last-child) {
+        border-bottom: 1px solid $filters-border;
```
Comment (spasovski) [inline src/media/css/app-list.styl]: Probably not necessary after my PR. We can either nuke it here or I'll resolve the conflict.

**Comment by spasovski** [comment]
Mega collections need a bottom padding so the border doesn't touch the bottom edge.

**Comment by spasovski** [comment]
Looks good r+wc sorry for the late review

**Comment by ngokevin** [comment]
thanks. i added clearfix to the app lists to give them bottom padding

**Pull request merged**